### PR TITLE
Add example for pushing metrics directly to Stackdriver

### PR DESCRIPTION
**Pull request by bskiba**
You can view the readme here: https://github.com/bskiba/k8s-stackdriver/blob/example-direct/custom-metrics-stackdriver-adapter/examples/direct-to-sd/README.md

### CODE REVIEW

**Comment by MaciekPytel** [comment]
/lgtm
From my point of view looks good, though @kawych's opinion is probably more important here.

**Comment by bskiba** [comment]
@kawych can you take a look?

File: custom-metrics-stackdriver-adapter/examples/direct-to-sd/README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+## Stackdriver dummy exporter
+
+A simple sd-dummy-exporter container exports a metric of constant value 
+to Stackdriver in a loop. The metric name and value can be passed in via flags.
+Pod id is passed to the container via downward API (see [custom-metrics-sd deployment]
+for how it's done)
+
+## Horizontal Pod Autoscaling object
+
+In the example, there is a [Horizontal Pod Autoscaler object] configured to scale the deployment on
```
Comment (kawych) [inline custom-metrics-stackdriver-adapter/examples/direct-to-sd/README.md]: Nit: please remove the repetition ("configured").
E.g. "In the example, a [Horizontal Pod Autoscaler object] is configured to scale the deployment on the exported metric with a target value lower than the default metric value."

Comment (bskiba) [reply custom-metrics-stackdriver-adapter/examples/direct-to-sd/README.md]: Done

**Pull request merged**